build/ci/.azure-devops-package-generic.yml

```diff
@@ -83,6 +83,10 @@ jobs:
       logProjectEvents: false
       createLogFile: false
 
+  - bash: |
+      dotnet tool install --global Basic.CompilerLog
+      complog create --projectName=Uno.UI $(build.artifactstagingdirectory)\build-$(GitVersion.FullSemVer)-$(XAML_FLAVOR_BUILD).binlog
```

Comment:
Let's see if this works and if I'll be able to get something useful from it. It's possible to turn this into a Microsoft.CodeAnalysis.Compilation, which could make investigation much easier!